sw/device/silicon_creator/lib/drivers/spi_device.h

```diff
@@ -0,0 +1,67 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+#ifndef OPENTITAN_SW_DEVICE_SILICON_CREATOR_LIB_DRIVERS_SPI_DEVICE_H_
+#define OPENTITAN_SW_DEVICE_SILICON_CREATOR_LIB_DRIVERS_SPI_DEVICE_H_
+
+#include <stddef.h>
+#include <stdint.h>
+
+#include "sw/device/silicon_creator/lib/error.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * Bit field definitions for JEDEC device ID.
+ *
+ * Note: spi_device sends bits 7:0 of the device ID first.
+ */
+#define SPI_DEVICE_DEV_ID_ROM_BOOTSTRAP_BIT 0
+#define SPI_DEVICE_DEV_ID_CHIP_REV_FIELD \
+  ((bitfield_field32_t){.mask = 0x7, .index = 1})
+#define SPI_DEVICE_DEV_ID_CHIP_GEN_FIELD \
+  ((bitfield_field32_t){.mask = 0xf, .index = 4})
+#define SPI_DEVICE_DEV_ID_DENSITY_FIELD \
+  ((bitfield_field32_t){.mask = 0xf, .index = 8})
```

Comment:
Mask = 0xFF, correct?  The density is an 8-bit field.

Reply:
Yes, density's mask should be 0xff, thanks!